private/buf/bufcli/bufcli.go

```diff
@@ -684,13 +685,20 @@ func promptUser(container app.Container, prompt string, isPassword bool) (string
 		if isPassword {
 			data, err := term.ReadPassword(int(file.Fd()))
 			if err != nil {
+				if errors.Is(err, io.EOF) {
+					return "", err
+				}
 				return "", NewInternalError(err)
 			}
 			value = string(data)
 		} else {
 			scanner := bufio.NewScanner(container.Stdin())
 			if !scanner.Scan() {
-				return "", NewInternalError(scanner.Err())
+				// scanner.Err() returns nil on EOF.
```

Comment:
Does it *only* return nil on `io.EOF`?

Reply:
Good question!

Yes, I believe nil is only possible on `io.EOF` or if the scanner's split func returns `bufio.ErrFinalToken`, which the default split func (`bufio.ScanLines`, which we're using here) never returns.

From https://pkg.go.dev/bufio#Scanner.Scan
> After Scan returns false, the Err method will return any error that occurred during scanning, except that if it was io.EOF, Err will return nil.